tests/spdx3/model/test_external_identifier.py

```diff
@@ -1,23 +1,31 @@
 # SPDX-FileCopyrightText: 2023 spdx contributors
 #
 # SPDX-License-Identifier: Apache-2.0
+from unittest import TestCase
+
 import pytest
 
 from spdx_tools.spdx3.model import ExternalIdentifier, ExternalIdentifierType
 
 
 def test_correct_initialization():
     external_identifier = ExternalIdentifier(
-        ExternalIdentifierType.CPE22, "cpe:/o:canonical:ubuntu_linux:10.04:-:lts", "This is a comment"
+        ExternalIdentifierType.CPE22,
+        "cpe:/o:canonical:ubuntu_linux:10.04:-:lts",
+        "This is a comment",
+        ["first locator", "second locator"],
+        "authority",
     )
     assert external_identifier.external_identifier_type == ExternalIdentifierType.CPE22
     assert external_identifier.identifier == "cpe:/o:canonical:ubuntu_linux:10.04:-:lts"
     assert external_identifier.comment == "This is a comment"
+    TestCase().assertCountEqual(external_identifier.identifier_locator, ["first locator", "second locator"])
```

Comment:
I don't think Python messes up the order of lists between setting and retrieving it.
```suggestion
    assert external_identifier.identifier_locator == ["first locator", "second locator"]
```